docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -25,6 +25,7 @@ spec:
   host: cafe.example.com
   tls:
     secret: cafe-secret
+  dos: defalt/dos-protected
```

Comment:
because dos is not the primary use case and it is also a paid addon, I don't think we should it include in in the main VirtuaServer YAML example in this doc. I think the added documentation and the GitHub examples cover it enough.

Reply:
fixed